recipes/pgscatalog.core/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "pgscatalog.core" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pgscatalog_core-{{ version }}.tar.gz
+  sha256: 1e14738f5f15fb067faa9cc321623e4269d6f505eb95d26f2b5ae9686e5bfb5e
+
+build:
+  entry_points:
+    - pgscatalog-download = pgscatalog.core.cli.download_cli:run
+    - pgscatalog-combine = pgscatalog.core.cli.combine_cli:run
+    - pgscatalog-relabel = pgscatalog.core.cli.relabel_cli:run
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  run_exports:  
+    - {{ pin_subpackage('pgscatalog.core', max_pin="x.x") }}  
+
+requirements:
+  host:
+    - python >=3.11,<4.0
```

Comment:
AFAIK support for Python 3.11 & 3.12 is available only in the `bulk` branch for now.
In `master` branch the available Python versions are 3.8, 3.9 and 3.10.
I wonder how this worked here.
The logs show that Python 3.12 has been installed - `python:           3.12.3-hab00c5b_0_cpython`

Reply:
Thanks, I've downgraded the minimum version to 3.10 in a patch release

When I check the new logs and artifacts they're being built with 3.12 still - will the latest commit be OK to merge or do I have to put an upper limit on the python version range?